src/EnergyPlus/VentilatedSlab.cc

```diff
@@ -337,26 +322,27 @@ namespace VentilatedSlab {
             state.dataVentilatedSlab->VentSlabNumericFields(Item).FieldNames.allocate(NumNumbers);
             state.dataVentilatedSlab->VentSlabNumericFields(Item).FieldNames = cNumericFields;
             UtilityRoutines::IsNameEmpty(state, state.dataIPShortCut->cAlphaArgs(1), CurrentModuleObject, ErrorsFound);
+            auto & ventSlab = state.dataVentilatedSlab->VentSlab(Item);
 
-            state.dataVentilatedSlab->VentSlab(Item).Name = state.dataIPShortCut->cAlphaArgs(1);
-            state.dataVentilatedSlab->VentSlab(Item).SchedName = state.dataIPShortCut->cAlphaArgs(2);
+            ventSlab.Name = state.dataIPShortCut->cAlphaArgs(1);
+            ventSlab.SchedName = state.dataIPShortCut->cAlphaArgs(2);
```

Comment:
I'm going to throw a few things out there and see what sticks. Thing 1. Do we really need to store `SchedName`? We immediately convert it to `SchedPtr` (Actually `SchedIdx` or `SchedNum`, we're not actually using pointers here). This is almost exactly the same thing as not storing the string of a Type.